src/tests/system/tests/test_passkey.py

```diff
@@ -0,0 +1,480 @@
+    """
+    :title: Check authentication of user with IPA and LDAP when server is not reachable
+    :setup:
+        1. Add a IPA and LDAP user with passkey_mapping.
+        2. Setup IDM client with FIDO and umockdev setup
+        3. Create the recording files when authentications is successful
+        4. Add 'pam_passkey_auth = True' under pam section of sssd.conf file.
+    :steps:
+        1. Check the umockdev-run command for authentication of the user.
+        2. Check umockdev-run that the output contains the username.
+        3. Make the server offline by updating (putting invalid) hostname of ipa server.
+        4. Check authentication of user again.
+    :expectedresults:
+        1. Command returns 0.
+        2. Output contains username of the user.
+        3. Server is offline.
+        4. Authentication works, getting username in output.
+    :customerscenario: False
+    """
+    tc = passkey_str = username = server_url = ""
+    if isinstance(provider, IPA):
+        tc = "su_ipa_test3"
+        username = 'user1@ipa.test'
+        passkey_str = "passkey:NUZMRUXIb/W8Ij1GqwCDHSCWxt/SxWxckwtQjLYi/X6Y1qZFB+HI8WO6khzAjzsz248" \
+                      "kHbaeAf9qfmqfCky1Jg==,MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEIasAa8ogjPCKXeA4K" \
+                      "Y3t0W3xBRmG+E4D+MNoRIAJrYuNLSYtAcOL7DCbIfgc+7c5Y4Mh/FzoEyeumKGYMoyTfg=="
+        server_url = "ipa_server"
+
+    elif isinstance(provider, LDAP):
+        tc = "su_ldap_test6"
+        username = "user1@test"
+        passkey_str = "passkey:mQEUTWdtDJPELQNTDdxXNHlfIO1qXFf0LVZjWEfyDALFzvLZ4e4XD5bemqq+o3ThrzT6k1I1n3Z2N00G" \
+                      "vLSmjQ==,MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEqk7K5VAI7Evr4ar8X82L/sxm/Bnm5Ti31xnLfGO0BipwH" \
+                      "ucw8+/wT4+6T9j5gdMwZKUcXR4BILpmULEyrcZUfw=="
+        server_url = "ldap_uri"
+
+    user = provider.user('user1').add()
+    user.passkey_add(passkey_str)
+
+    client.sssd.pam['pam_passkey_auth'] = 'true'
+    client.sssd.start()
+
+    for record_file in [f'{tc}.ioctl', f'{tc}.script', f'{tc}.device']:
+        src_file = f"{os.getcwd()}/tests/data/passkey/{tc}/{record_file}"
+        client.tools.copy_files(src_file, f"/tmp/{record_file}")
+
+    output1 = client.auth.passkey.su(tc, username, 123456)
+
+    # Here we are making server/backend offline but not deleting cache and logs.
+
+    client.sssd.config.remove_option('domain/test', server_url)
+    client.sssd.domain[server_url] = "ldap://new.server.test"
+    client.sssd.start()
+
+    output2 = client.auth.passkey.su(tc, username, 123456)
+
+    assert output1.stdout_lines[-1] == output2.stdout_lines[-1] == 'user1'
+
+
+@pytest.mark.tier(1)
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_passkey__su_check_when_srv_offline_firewalld(client: Client, provider: GenericProvider):
+    """
+    :title: Check authentication of user with LDAP and IP when server is not reachable by removing ldap
```

Comment:
Should IP there be IPA?  

I would also specifically mention "server port is not reachable".    IMO you can drop the "by removing ldap and ldaps port" from the title.  That's more an implementation detail I think.